src/workflows/workflows.ts

```diff
@@ -364,10 +362,9 @@ export class Workflows {
   }
 
   public createTradeWithSigner(
-    l1Signer: L1Signer,
-    l2Signer: L2Signer,
+    walletConnection: WalletConnection,
     request: GetSignableTradeRequest,
   ) {
-    return createTradeWorkflowWithSigner(l1Signer, l2Signer, request, this.tradesApi);
+    return createTradeWorkflowWithSigner(walletConnection, request, this.tradesApi);
```

Comment:
to align with the other function, we should ramp the params into the object
{l1Signer, l2Signer, request, tradeApi: this.tradeAPI}